rotkehlchen/assets/utils.py

```diff
@@ -86,6 +98,18 @@ def get_or_create_evm_token(
             if None in (decimals, symbol, name):
                 raise NotERC20Conformant(f'Token {evm_address} is not ERC20 conformant')  # noqa: E501  # pylint: disable=raise-missing-from
 
+        if len(info) == 0 and asset_exists is True:
```

Comment:
I think that this check makes no sense due to:
1. `ethereum_manager.get_basic_contract_info` never returns an empty dictionary. If it doesn't find anything, it returns a dictionary with `None` values. So `len(info)` will never happen.
2. If some properties (name or symbol or decimals) are missing, then it will be caught above and `NotERC20Conformant` will be raised

Reply:
I think you are right. 

Perhaps also modify the docstring of `get_basic_contract_info()` from `Although the values might be empty.` to `Although the values might be None` to avoid any such future misunderstandings?